### DISCUSSION TITLE
npmjs.org: Will 0.31.1 be published to npm? 😇

### DISCUSSION THREAD

**Discussion by antonhedstrom** [react-recompose/react-recompose#12] [Q&A] [answered]
Would appreciate if 0.31.1 will be published to npm so I can install using `npm install`. I would like to take part of the React.createFactory (introduced in 0.31.0) fix which currently bloats our test result. 

Thanks!

<img width="573" alt="Screenshot 2021-04-27 at 10 20 07" src="https://user-images.githubusercontent.com/506266/116209397-2972c680-a742-11eb-8bd4-5a5b40c3d8f3.png">

**Comment by brody2consult** [comment, marked as answer]
Hello I did already publish `0.31.1`. Please be sure to use the correct npm package from here: https://www.npmjs.com/package/react-recompose

My request for publishing rights to the original `recompose` package was blocked due to this: https://github.com/npm/roadmap/issues/28

**Reply by antonhedstrom** [reply]
Oh, sorry and thanks for clarification! 
We will probably switch to be using this package instead then. Thanks!
